projects/packages/tracking/src/class-tracking.php

```diff
@@ -87,6 +87,19 @@ public function ajax_tracks() {
 		wp_send_json_success();
 	}
 
+	/**
+	 * Register script necessary for tracking.
+	 */
+	public function register_tracks_scripts() {
+		wp_register_script(
+			'jp-tracks-functions',
+			Assets::get_file_url_for_environment( 'js/tracks-callables.js', 'js/tracks-callables.js', __FILE__ ),
+			array( 'jp-tracks' ),
```

Comment:
what is jp-tracks?

The track package has jptracks. Is there duplicate or multiple versions. could be a bigger item to untangle.

Reply:
wp_enqueue_script(
			'jp-tracks',
			'//stats.wp.com/w.js',
			array(),
			gmdate( 'YW' ),
			true
		);